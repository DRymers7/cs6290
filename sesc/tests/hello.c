#include <stdio.h>

int main(int argc, char *argv[]){
	printf("Hi! I am Derek Rymers and this string is more than 30 characters!!!!\n");
	return 0;
}
