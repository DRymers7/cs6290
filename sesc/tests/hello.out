Hi! I am Derek Rymers!!!
